projects/aws/upgrader/cmd/upgrade_node.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/aws/eks-anywhere-build-tooling/projects/aws/upgrader/upgrade"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var upgradeNodeCmd = &cobra.Command{
+	Use:   "node",
+	Short: "Upgrade node",
+	Long:  "Use InPlace Upgrader upgrade node to upgrade kubeadm on the node",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var err error
+
+		nodeType := viper.GetString("type")
+		k8sVersion := viper.GetString("k8sVersion")
+		etcdVersion := viper.GetString("etcdVersion")
+		if etcdVersion == "" {
+			etcdVersion = "NO_UPDATE"
+		}
+
+		err = upgradeNode(cmd.Context(), nodeType, k8sVersion, etcdVersion)
+		if err != nil {
+			log.Fatalf("upgrade node failed: %v", err)
+		}
+		return nil
+	},
+}
+
+func init() {
+	var err error
+
+	upgradeCmd.AddCommand(upgradeNodeCmd)
+	upgradeNodeCmd.Flags().String("type", "", "node type flag")
```

Comment:
Pass this in as a flag instead of as a string? Do we anticipate significantly more here?

Reply:
with flags we could have some basic validations but with args as string it's harder to validate. is that what you are asking.?